src/components/pdf/PdfPage4.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import PdfFooter from '@/components/pdf/PdfFooter.vue'
+import PdfHeader from '@/components/pdf/PdfHeader.vue'
+import type { AutocalsolResult as AutocalsolResultType } from '@/model/autocalsol.model'
+import AutocalsolResultGraph from '@/components/results/AutocalsolResultGraph.vue'
+
+const props = defineProps<{
+  autocalsolResult: AutocalsolResultType
+}>()
+</script>
+
+<template>
+  <div class="w-[90%] font-dm-sans">
+    <!-- Header -->
+    <PdfHeader />
+
```

Comment:
<div class="flex flex-row items-center gap-2">
        <img :src="economies" alt="" class="w-11 h-10" />
        <span class="font-bold text-2xl"> Votre production d'énergie </span>
</div>

is missing before graph